opencraft/settings.py

```diff
@@ -586,9 +586,6 @@
 
 # Ansible #####################################################################
 
-# Ansible requires a Python 2 interpreter
-ANSIBLE_PYTHON_PATH = env('ANSIBLE_PYTHON_PATH', default='/usr/bin/python')
```

Comment:
If I left that, the new check would never trigger because `settings.ANSIBLE_PYTHON_PATH` is always defined.
I could check if the value is `python` instead, but then that would trigger when the user explicitly set it to that value.

The idea is that, in a "production" setting, the user "should" define the value, even if it's just to say "I know what I'm doing".

Reply:
The only way we configure this is via environment variables, so removing this means we can't change it at all. 

What you can do instead is set the default to `None`, and the check can see if it's not none.

Reply:
🤦 right, thanks for the heads-up!